DiscordBot/Models/GuildUser.cs

```diff
@@ -11,6 +11,6 @@ public class GuildUser
 
         public Guild Guild { get; set; }
 
-        public DateTimeOffset DateJoined { get; set; }
+        public DateTimeOffset? DateJoined { get; set; }
```

Comment:
Out of curiosity, what is the advantage of DateJoined being nullable? Is there a scenario where that information wouldn't be available for a user?

Reply:
Good point...I usually just set most or all my DateTimes to nullable on the off chance I need it to be null...but you're right there's no reason for that.  I can't really think of a scenario where we wouldn't have the date joined date.